Plugins/NativeCamera/Source/Apple/CameraDevice.mm

```diff
@@ -3,6 +3,7 @@
 #endif
 
 #import <MetalKit/MetalKit.h>
+//#import <Photos/Photos.h>
```

Comment:
?

Reply:
This was in conjunction with a comment later in the file, but you're right, if someone sees this here they aren't going to know it goes with the comment later and just delete. I will remove it and update the comment to just say to add this header instead of uncommenting it.